mmv1/third_party/terraform/resources/resource_compute_instance.go.erb

```diff
@@ -1260,11 +1258,9 @@ func resourceComputeInstanceRead(d *schema.ResourceData, meta interface{}) error
 	if err := d.Set("machine_type", tpgresource.GetResourceNameFromSelfLink(instance.MachineType)); err != nil {
 		return fmt.Errorf("Error setting machine_type: %s", err)
 	}
-<% unless version == 'ga' -%>
```

Comment:
I searched in the file and it looks like there are couple more lines to be removed, could you search `networkPerformanceConfig` in this file to clean them up?

Line 1069, 1095